### ISSUE TITLE
Support for CSS Sources in Compile Includes

### ISSUE THREAD

**Issue by ebollens** [ucla-oarc-web/WebBlocks#95]
Right now, the build only supports `scss` files for inclusion. This should be changed to include `css` files as well, as they should be compilable as part of Compass' build process. This is necessary for adapters to frameworks that aren't written in SASS.

**Comment by ebollens** [comment]
Tried to implement it but found it not possible because of the way SASS works: the short answer is that the semantics of `@import` have been defined as different for `.scss` files than for `.css` files, but because SASS is a superset of CSS, they cannot change the semantics of `@import` for `.css` files. The full discussion is here: https://github.com/nex3/sass/issues/193

That said, the best I can do here is provide another folder `/src/css` that will suck these CSS files in directly as appended contents. Maybe I can implement the same thing for adapter CSS includes too, although I'm not exactly sure why an adapter would toy with CSS files (yuck!)

**Comment by ebollens** [comment]
I decided to provide support for appended CSS sources, but _not_ to support appending CSS from adapters. 

From the high level, there is a logical discrepancy in that adapters are supposed to define mixins and variables, and CSS isn't this. At the low level, there is also a problem with the load stack if we wanted to do this in that CSS is appended after the Compass compiler runs, and thus you'd have adapter CSS definitions defined later in your build file, which would create a problem in that you'd be overriding things from WebBlocks with CSS even if you specified rules in the SASS that should logically override the adapter definitions.

That said, this is now resolved with c782588.
